src/EVM.hs

```diff
@@ -1010,7 +1011,7 @@ exec1 conf = do
                 let cost = if acc then 0 else g_cold_account_access
                     funds = this.balance
                     recipientExists = accountExists xTo vm
-                branch (Expr.iszero $ Expr.eq funds (Lit 0)) $ \hasFunds -> do
+                branch (?conf.maxExplore) (Expr.iszero $ Expr.eq funds (Lit 0)) $ \hasFunds -> do
```

Comment:
What's `?conf.maxEplore`? What does the question mark mean?

Reply:
I guess this is the implicit parameter thing, right?

Reply:
Yes! It's the implicit parameter thing. Kinda annoying, but easier than passing `maxExplore` everywhere else...